client/src/modules/shopping-list/model/reducer.js

```diff
@@ -1,5 +1,9 @@
 import { items as itemsInitialState, initialStatus } from './initialState';
-import { FETCH_FAILED, FETCH_ITEMS } from 'modules/shopping-list/model/actions';
+import {
+  FETCH_FAILED,
+  FETCH_ITEMS,
+  ADD_SHOPPING_LIST_SUCCESS
+} from './actions';
```

Comment:
action should be enum and imported as `ShoppingListActionTypes`. Currently in code we have no idea what action `FETCH_FAILED' really mean.